### PR TITLE
remove debug mode

### PR DESCRIPTION
**Pull request by essayharpereal**
[main.cpp.txt](https://github.com/C3RV1/UndertaleNDS/files/11711437/main.cpp.txt)

### CODE REVIEW

File: source/main.cpp [hunk 1]
```diff
@@ -62,7 +59,7 @@ int main() {
                 delete globalCutscene;
                 globalCutscene = nullptr;
                 globalInGameMenu.show(false);
-                globalPlayer->setPlayerControl(true);
```
Comment (C3RV1) [inline source/main.cpp:65]: why this change? I was about to merge this

Comment (C3RV1) [reply source/main.cpp:65]: actually this wouldn't complie: _playerControl is private

**Review by C3RV1 — CHANGES_REQUESTED** [review]
change back the line relating to _playerControl

File: source/main.cpp (hunk 1 above)
Comment (essayharper) [reply source/main.cpp:65]: This was in the original code at the time

**Comment by C3RV1** [comment]
Already removed debug variables from main.cpp
